pkgs/os-specific/darwin/apple-source-releases/system_cmds/meson.build

```diff
@@ -0,0 +1,406 @@
+# Project settings
```

Comment:
where did this file come from? did you created it or is it downloaded from somewhere and edited?

Reply:
I created it. The latest force push indicates where I got the settings (from the upstream Xcode project).

Reply:
> do we need the commented out blocks?

I included them to document why certain commands weren’t being built.